src/main/java/frc/robot/commands/DriveForwardTillTrueCommand.java

```diff
@@ -18,7 +18,7 @@
 public class DriveForwardTillTrueCommand extends EntechCommandBase {
```

Comment:
I am not a big fan of how this command is implemented.  The end condition is hidden outside the command, and there are no parameters for the distance to travel, speed, direction, etc.  How do we think this command will be used?  
We also need to fix the spelling Till -> Until
It's not worth holding up this merge for these comments though.

Reply:
I agree.  a student implemented this and I didnt want to erase it, but I agree that simple paraemters as inputs is much better.

I am going to leave this as-is, but i will mark a TODO: to address this